portfolio/src/main/webapp/style.css

```diff
@@ -28,9 +28,33 @@ body {
     float: right;
 }
 
+.travel-pic {
```

Comment:
Recommend using consistent indentation, likely 2 spaces.

I wonder if your editor might be auto-formatting some of these to be inconsistent.